src/Symfony/Component/Console/Tests/Tester/CommandTesterTest.php

```diff
@@ -267,4 +269,23 @@ public function testErrorOutput()
 
         $this->assertSame('foo', $tester->getErrorOutput());
     }
+
+    public function testAInvokableExtendedCommand()
+    {
+        $command = new SomeInvokableExtendedTestCommand();
+
+        $tester = new CommandTester($command);
+        $tester->execute([]);
+
+        $tester->assertCommandIsSuccessful();
+    }
+}
+
+#[AsCommand(name: 'invokable-extended')]
+class SomeInvokableExtendedTestCommand extends Command
+{
+    public function __invoke(): int
+    {
+        return 0;
+    }
```

Comment:
This case was already working, as this class extends `Command`.

The new feature test would use `InvokableTestCommand`:
```suggestion

    public function testInvokableCommand()
    {
        $command = new InvokableTestCommand();

        $tester = new CommandTester($command);
        $tester->execute([]);

        $tester->assertCommandIsSuccessful();
    }
}
```